src/sidebar/components/test/thread-list-test.js

```diff
@@ -162,23 +156,16 @@ describe('ThreadList', () => {
   });
 
   context('active scroll to an annotation thread', () => {
-    let stubbedDocument;
-    let stubbedFakeScrollContainer;
     let fakeScrollTop;
 
     beforeEach(() => {
       fakeScrollTop = sinon.stub();
-      stubbedFakeScrollContainer = sinon
-        .stub(fakeScrollContainer, 'scrollTop')
-        .set(fakeScrollTop);
-      stubbedDocument = sinon
-        .stub(document, 'querySelector')
-        .returns(fakeScrollContainer);
+      sinon.stub(fakeScrollContainer, 'scrollTop').set(fakeScrollTop);
+      sinon.stub(document, 'querySelector').returns(fakeScrollContainer);
```

Comment:
I'd recommend using `sinon.stub(...).withArgs(...)` here so that `querySelector` only returns the scroll container if passed the expected selector.